sdgym/benchmark.py

```diff
@@ -392,7 +411,7 @@ def run(synthesizers, datasets=None, datasets_path=None, modalities=None, bucket
         raise SDGymError("No valid Dataset/Synthesizer combination given")
 
     scores = pd.concat(scores)
-    if output_path:
-        scores.to_csv(output_path, index=False)
+
+    _write_output_to_csv(scores, output_path, aws_key, aws_secret)
```

Comment:
After comments from #77 are addressed this can become a one line change to `write_csv(scores, output_path)`